engine/Assets/Scripts/UI/Dynamic/Modals/Configuring/ConfigMotorModal.cs

```diff
@@ -0,0 +1,240 @@
+                .SetRightStretch<ScrollView>()
+                .SetTopStretch<ScrollView>(0, 0, -nameLabelContent.Parent!.RectOfChildren().yMin + PADDING * 2)
+                .SetHeight<ScrollView>(MODAL_HEIGHT - nameLabelContent.Parent!.RectOfChildren().yMin - 50);
+
+        _scrollViewWidth = _scrollView.Parent!.RectOfChildren().width - SCROLL_WIDTH;
+
+        CreateEntry("Drive", (_motors[0].driver as WheelDriver).Motor.targetVelocity, x => ChangeDriveVelocity(x));
+        if (_robotISSwerve) {
+            CreateEntry(
+                "Turn", (_motors[driveMotorCount].driver as RotationalDriver).Motor.targetVelocity, x => ChangeTurnVelocity(x));
+        }
+
+        // original target velocities for each motor and entry in scrollview for other motors
+        for (i = 0; i < _motors.Length; i++) {
+            var driver = _motors[i].driver;
+            switch (driver) {
+                case (RotationalDriver):
+                    _motors[i].origVel = (driver as RotationalDriver).Motor.targetVelocity;
+                    break;
+                case (WheelDriver):
+                    _motors[i].origVel = (driver as WheelDriver).Motor.targetVelocity;
+                    break;
+            }
+
+            if (_motors[i].motorType == MotorType.Other) {
+                int j = i;
+                CreateEntry(i.ToString(), _motors[j].origVel, x => _motors[j].setTargetVelocity(x));
+            }
+        }
+    }
+
+    public override void Update() {}
+
+    public override void Delete() {}
+
+    private void CreateEntry(string name, float currVel, Action<float> onClick) {
+        (Content nameContent, Content velContent) =
+            _scrollView.Content.CreateSubContent(new Vector2(_scrollViewWidth, 40f))
+                .SetTopStretch<Content>(0, 0, 0)
+                .ApplyTemplate<Content>(VerticalLayout)
+                .SplitLeftRight(NAME_WIDTH, PADDING);
+        nameContent.CreateLabel().SetText(name).SetTopStretch(0, PADDING, PADDING + _scrollView.HeightOfChildren);
+        velContent.CreateSlider("", minValue: 0f, maxValue: 150f, currentValue: currVel)
+            .SetTopStretch<Slider>(PADDING, PADDING, _scrollView.HeightOfChildren)
+            .AddOnValueChangedEvent((s, v) => { onClick(v); });
+    }
+
+    private void ChangeDriveVelocity(float vel) {
+        foreach (ConfigMotor motor in _motors) {
+            if (motor.motorType == MotorType.Drive)
+                motor.setTargetVelocity(vel);
+        }
+    }
+
+    private void ChangeTurnVelocity(float vel) {
+        foreach (ConfigMotor motor in _motors) {
+            if (motor.motorType == MotorType.Turn)
+                motor.setTargetVelocity(vel);
+        }
+    }
+
+    private class ConfigMotor {
+        public Driver driver { get; set; }
+        public float origVel { get; set; }
+        private float _force { get; set; }
+        public bool velChanged { get; set; } = false;
+        public MotorType motorType { get; set; }
```

Comment:
What's the reason for having `{ get; set; }` after each of these? I don't think it's doing anything in this case.